tests/test_abandon_command.rs

```diff
@@ -56,11 +56,11 @@ fn test_rebase_branch_with_merge() {
 
     let stdout = test_env.jj_cmd_success(&repo_path, &["abandon", "d"]);
     insta::assert_snapshot!(stdout, @r###"
-    Abandoned commit b7c62f28ed10 d
+    Abandoned commit vruxwmqv b7c62f28 d
     Rebased 1 descendant commits onto parents of abandoned commits
-    Working copy now at: 11a2e10edf4e e
-    Parent commit      : 2443ea76b0b1 a
-    Parent commit      : fe2e8e8b50b3 c
+    Working copy now at: znkkpsqq 11a2e10e e
+    Parent commit      : rlvkpnrz 2443ea76 a
+    Parent commit      : royxmykx fe2e8e8b c
     Added 0 files, modified 0 files, removed 1 files
```

Comment:
Another, more straightforward, TODO: `??` for divergent change ids.

Reply:
This turns out to *also* cause problems, since some functions call `short_commit_template` before they do `rebase_descendants`. I'll likely show the problem in a draft PR if I can't figure it out quickly. In any case, this is also postponed to another PR.